src/esssans/normalization.py

```diff
@@ -152,17 +159,22 @@ def iofq_denominator(
     # solid_angle (pixel-dependent) and monitors (wavelength-dependent) will fail.
     # The direct beam may also be pixel-dependent. In this case we need to drop
     # variances of the other term already before multiplying by solid_angle.
-    broadcast = {
-        UncertaintyBroadcastMode.drop: drop_variances_if_broadcast,
-        UncertaintyBroadcastMode.upper_bound: broadcast_with_upper_bound_variances,
-        UncertaintyBroadcastMode.fail: lambda x, sizes: x,
-    }[uncertainties]
+    broadcast = _broadcasters[uncertainties]
     if direct_beam is not None:
         denominator = direct_beam * broadcast(denominator, sizes=direct_beam.sizes)
```

Comment:
```suggestion
    if direct_beam is not None:
        denominator = direct_beam * _broadcasters[uncertainties](denominator, sizes=direct_beam.sizes)
```

Reply:
Not sure I see the difference? If it is just style, it will probably be longer or harder to read since black will add weird linebreaks.

Reply:
Just thought it removed the need to create the `broadcast` variable outside of the `if direct_beam is not None:`.
Could also be
```Py
    if direct_beam is not None:
        broadcast = _broadcasters[uncertainties]
        denominator = direct_beam * broadcast(denominator, sizes=direct_beam.sizes)
```
?

I really don't mind, it was a suggestion. As you say, it's mostly style.

Sometimes I wish github had two types of suggestions: one where you actually would like to see the change made, and another which would be more optional, an actual "suggestion", as in "take it or leave it"

Reply:
In Python this does not make a difference anyway, this is not C(++) scoping rules.